packages/account/src/Components/poa/common-mistakes/common-mistake-examples.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import IcDocumentNameMismatch from '../../../Assets/ic-document-name-mismatch.svg';
+import IcDocumentAddressMismatch from '../../../Assets/ic-document-address-mismatch.svg';
+import IcOldIssuedDocument from '../../../Assets/ic-old-issued-document.svg';
+import IcBlurryDocument from '../../../Assets/ic-blurry-document.svg';
+import IcCroppedDocument from '../../../Assets/ic-cropped-document.svg';
+import IcEnvelope from '../../../Assets/ic-envelop.svg';
+import IcErrorBadge from '../../../Assets/ic-error-badge.svg';
+import './common-mistake-examples.scss';
+
+type TCommonMistakeExamplePartialsProps = {
+    description: string;
+    image: React.ReactElement;
+};
+
+/**
+ * Wrapper for displaying the image and description of the mistake
+ * @param {string} description - description of the mistake
+ * @param {React.ReactElement} image - Sample image
+ * @returns  {React.ReactElement} - React element
+ */
+const CommonMistakeExamplePartials = ({ description, image }: TCommonMistakeExamplePartialsProps) => (
+    <div className='layout'>
+        {image}
+        <div className='description'>
```

Comment:
CommonMistakeExamplePartials is specific to `CommonMistakeExamples` so you should define the classNames in a way to properly follow the standard e.g.  `common-mistake-examples__content-layout`